public/blog/testrpc-is-now-ganache.md

```diff
@@ -0,0 +1,35 @@
+# TestRPC is now Ganache
+
+Have you been looking for the TestRPC recently and haven't been able to find it?
+
+Don't worry, it's still here! It's just now known as **Ganache**.
+
+It comes in two flavors: A fully-interactive graphical interface, and the more familiar command-line version.
+
+* [Get the graphical version here](/ganache).
+
+* For the command-line version (known as **Ganache CLI**):
+
+  ```shell
+  npm install -g ganache-cli
+  ```
+
+## A little history
+
+Prior to the existence of the TestRPC, you would have to deploy to private internal test networks to see how your contract code operated (as public test networks like Ropsten didn't yet exist!).
+
+While initially under the auspices of the [EthereumJS project](https://github.com/ethereumjs/), it was originally created by the Truffle team, and today we remain its primary maintainers.
+
+So simply for maintainability sake, it just made sense to bring the TestRPC under Truffle's wing.
```

Comment:
My gut says this and the two paragraphs above it should be one paragraph, mostly because of the shared "it. I got confused what the it was for a second.